vhdbuilder/packer/kube-proxy-images.json

```diff
@@ -52,19 +33,12 @@
       {
         "downloadURL": "mcr.microsoft.com/oss/kubernetes/kube-proxy:v*",
         "versions": [
-          "1.19.1-hotfix.20200923",
-          "1.19.1-hotfix.20200923.1",
+          "1.19.9-hotfix.20210526.2",
```

Comment:
Should the latest hotfix version `1.19.9-hotfix.20210526.3` be always used if the cx has updated the node image to the latest one? If so, we can only keep `1.19.9-hotfix.20210526.3`

Reply:
I believe so but I want to confirm before removing. Perhaps @palma21  or @qpetraroia can help answer the requirement based on AKS release policies/users' expectations?

Reply:
Should be fine to follow up on this

Reply:
Yes, that is the policy/expectation.